.buildkite/tests.sh

```diff
@@ -20,23 +42,19 @@ while ! docker ps; do
 done
 
 echo "Download latest Earthly binary"
-curl -o ./earthly-released -L https://github.com/earthly/earthly/releases/latest/download/earthly-"$EARTH_OS"-amd64 && chmod +x ./earthly-released
+if [ -n "$download_url" ]; then
+    curl -o ./earthly-released -L "$download_url" && chmod +x ./earthly-released
+    released_earthly=./earthly-released
+fi
 
 echo "Build latest earthly using released earthly"
-./earthly-released +for-"$EARTH_OS"
+"$released_earthly" +for-"$EARTHLY_OS"
 
 echo "Execute tests"
-./build/"$EARTH_OS"/amd64/earthly --ci -P +test
-
-# Temporarily disable until failure is addressed.
-#echo "Execute experimental tests"
-#./build/"$EARTH_OS"/amd64/earthly --ci -P ./examples/tests+experimental
+"$earthly" --ci -P +test
 
 echo "Execute fail test"
-bash -c "! ./build/$EARTH_OS/amd64/earthly --ci +test-fail"
-
-echo "Build examples1"
-./build/"$EARTH_OS"/amd64/earthly --ci -P +examples1
+bash -c "! $earthly --ci ./examples/tests/fail+test-fail"
 
-echo "Build examples2"
-./build/"$EARTH_OS"/amd64/earthly --ci -P +examples2
+echo "Build examples"
+"$earthly" --ci -P +examples
```

Comment:
I sure hope the path to the earthly binary doesn't contain any spaces; but good to know it'll still work if it did :grinning: